est/jcabi-simpledb.est

```diff
@@ -0,0 +1,17 @@
+date: 28-05-2015
+author: Jason Wong
+method: champions.pert
+scope:
+  1: jcabi-simpledb project scaffolding
+  2: Implementation of simpledb AWS SDK wrapper
+  3: Achieve 80% test coverage
+  4: CI integration
```

Comment:
@longtimeago Travis and Rultor configs are meant to be CI integration. I think this estimate should be thinking from ground up, which means back to the times when this project is newly built, what tasks do we need to do. That is why I put CI integration here as a task to make out the rultor and travis configs. It is not a major task however as you can see in the estimate file. Hope it makes sense. Thanks.